app/src/main/java/com/eveningoutpost/dexdrip/NFCReaderX.java

```diff
@@ -282,6 +309,10 @@ protected void onPostExecute(Tag tag) {
                 if (succeeded) {
                     long now = JoH.tsl();
                     String SensorSn = LibreUtils.decodeSerialNumberKey(tag.getId());
+                    
+                    if (SensorSanity.checkLibreSensorChangeIfEnabled(SensorSn)) {
+                        Log.e(TAG, "Problem with Libre Serial Number - not processing");
```

Comment:
This logs that it isn't processing but does processing continue? should it?

Reply:
Sensor was stopped, so even if processing continues,  BG will not be created.
I can add a return in all this casses.